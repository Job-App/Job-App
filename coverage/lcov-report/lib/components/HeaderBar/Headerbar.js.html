
<!doctype html>
<html lang="en">

<head>
    <title>Code coverage report for lib/components/HeaderBar/Headerbar.js</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../../prettify.css" />
    <link rel="stylesheet" href="../../../base.css" />
    <link rel="shortcut icon" type="image/x-icon" href="../../../favicon.png" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../../sort-arrow-sprite.png);
        }
    </style>
</head>
    
<body>
<div class='wrapper'>
    <div class='pad1'>
        <h1><a href="../../../index.html">All files</a> / <a href="index.html">lib/components/HeaderBar</a> Headerbar.js</h1>
        <div class='clearfix'>
            
            <div class='fl pad1y space-right2'>
                <span class="strong">100% </span>
                <span class="quiet">Statements</span>
                <span class='fraction'>13/13</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">100% </span>
                <span class="quiet">Branches</span>
                <span class='fraction'>0/0</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">100% </span>
                <span class="quiet">Functions</span>
                <span class='fraction'>1/1</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">100% </span>
                <span class="quiet">Lines</span>
                <span class='fraction'>1/1</span>
            </div>
        
            
        </div>
        <p class="quiet">
            Press <em>n</em> or <em>j</em> to go to the next uncovered block, <em>b</em>, <em>p</em> or <em>k</em> for the previous block.
        </p>
    </div>
    <div class='status-line high'></div>
    <pre><table class="coverage">
<tr><td class="line-count quiet"><a name='L1'></a><a href='#L1'>1</a></td><td class="line-coverage quiet"><span class="cline-any cline-yes">1x</span></td><td class="text"><pre class="prettyprint lang-js">var _interopRequireDefault=require("@babel/runtime/helpers/interopRequireDefault");Object.defineProperty(exports,"__esModule",{value:true});exports.default=void 0;var _react=_interopRequireDefault(require("react"));var _reactNative=require("react-native");var _this=this,_jsxFileName="C:\\Users\\guo\\Desktop\\EECS393\\Job-App\\src\\components\\HeaderBar\\Headerbar.js";var _Dimensions$get=_reactNative.Dimensions.get('screen'),width=_Dimensions$get.width,height=_Dimensions$get.height;var HeaderBar=function HeaderBar(){return _react.default.createElement(_reactNative.View,{style:styles.container,__self:_this,__source:{fileName:_jsxFileName,lineNumber:14,columnNumber:9}},_react.default.createElement(_reactNative.View,{style:[styles.buttonContainer,styles.buttonContainerLeft],__self:_this,__source:{fileName:_jsxFileName,lineNumber:15,columnNumber:13}},_react.default.createElement(_reactNative.TouchableOpacity,{activeOpacity:.8,style:styles.button,__self:_this,__source:{fileName:_jsxFileName,lineNumber:16,columnNumber:17}},_react.default.createElement(_reactNative.Text,{style:styles.plus,__self:_this,__source:{fileName:_jsxFileName,lineNumber:19,columnNumber:25}},"Add"))),_react.default.createElement(_reactNative.View,{style:[styles.buttonContainer,styles.buttonContainerRight],__self:_this,__source:{fileName:_jsxFileName,lineNumber:22,columnNumber:13}},_react.default.createElement(_reactNative.TouchableOpacity,{activeOpacity:.8,style:styles.button,__self:_this,__source:{fileName:_jsxFileName,lineNumber:23,columnNumber:17}},_react.default.createElement(_reactNative.Text,{style:styles.plus,__self:_this,__source:{fileName:_jsxFileName,lineNumber:26,columnNumber:25}},"Login"))));};var styles=_reactNative.StyleSheet.create({container:{width:width,height:height/12,backgroundColor:'#3D348B',flexDirection:'row'},buttonContainer:{flex:1,marginTop:'auto',marginBottom:10},buttonContainerLeft:{alignItems:'flex-start'},buttonContainerRight:{alignItems:'flex-end'},button:{justifyContent:'center',alignItems:'center',marginLeft:24,marginRight:24},plus:{color:'#E0E2DB',fontSize:24}});var _default=HeaderBar;exports.default=_default;</pre></td></tr></table></pre>

                <div class='push'></div><!-- for sticky footer -->
            </div><!-- /wrapper -->
            <div class='footer quiet pad2 space-top1 center small'>
                Code coverage generated by
                <a href="https://istanbul.js.org/" target="_blank">istanbul</a>
                at Fri Nov 06 2020 09:07:42 GMT-0800 (Pacific Standard Time)
            </div>
        </div>
        <script src="../../../prettify.js"></script>
        <script>
            window.onload = function () {
                prettyPrint();
            };
        </script>
        <script src="../../../sorter.js"></script>
        <script src="../../../block-navigation.js"></script>
    </body>
</html>
    