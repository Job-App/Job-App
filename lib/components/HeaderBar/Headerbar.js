var _interopRequireDefault=require("@babel/runtime/helpers/interopRequireDefault");Object.defineProperty(exports,"__esModule",{value:true});exports.default=void 0;var _react=_interopRequireDefault(require("react"));var _reactNative=require("react-native");var _this=this,_jsxFileName="C:\\Users\\guo\\Desktop\\EECS393\\Job-App\\src\\components\\HeaderBar\\Headerbar.js";var _Dimensions$get=_reactNative.Dimensions.get('screen'),width=_Dimensions$get.width,height=_Dimensions$get.height;var HeaderBar=function HeaderBar(){return _react.default.createElement(_reactNative.View,{style:styles.container,__self:_this,__source:{fileName:_jsxFileName,lineNumber:14,columnNumber:9}},_react.default.createElement(_reactNative.View,{style:[styles.buttonContainer,styles.buttonContainerLeft],__self:_this,__source:{fileName:_jsxFileName,lineNumber:15,columnNumber:13}},_react.default.createElement(_reactNative.TouchableOpacity,{activeOpacity:.8,style:styles.button,__self:_this,__source:{fileName:_jsxFileName,lineNumber:16,columnNumber:17}},_react.default.createElement(_reactNative.Text,{style:styles.plus,__self:_this,__source:{fileName:_jsxFileName,lineNumber:19,columnNumber:25}},"Add"))),_react.default.createElement(_reactNative.View,{style:[styles.buttonContainer,styles.buttonContainerRight],__self:_this,__source:{fileName:_jsxFileName,lineNumber:22,columnNumber:13}},_react.default.createElement(_reactNative.TouchableOpacity,{activeOpacity:.8,style:styles.button,__self:_this,__source:{fileName:_jsxFileName,lineNumber:23,columnNumber:17}},_react.default.createElement(_reactNative.Text,{style:styles.plus,__self:_this,__source:{fileName:_jsxFileName,lineNumber:26,columnNumber:25}},"Login"))));};var styles=_reactNative.StyleSheet.create({container:{width:width,height:height/12,backgroundColor:'#3D348B',flexDirection:'row'},buttonContainer:{flex:1,marginTop:'auto',marginBottom:10},buttonContainerLeft:{alignItems:'flex-start'},buttonContainerRight:{alignItems:'flex-end'},button:{justifyContent:'center',alignItems:'center',marginLeft:24,marginRight:24},plus:{color:'#E0E2DB',fontSize:24}});var _default=HeaderBar;exports.default=_default;