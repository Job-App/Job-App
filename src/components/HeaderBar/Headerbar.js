import React from 'react';
import {
  View,
  Text,
  TouchableOpacity,
  StyleSheet,
  Dimensions,
} from 'react-native';

const {width, height} = Dimensions.get('screen');

const HeaderBar = () => {
    return(
        <View style={styles.container}>
            <View style={[styles.buttonContainer,styles.buttonContainerLeft]}>
                <TouchableOpacity
                    activeOpacity={.8}
                    style={styles.button}>
                        <Text style={styles.plus}>Add</Text>
                </TouchableOpacity>
            </View>
            <View style={[styles.buttonContainer,styles.buttonContainerRight]}>
                <TouchableOpacity
                    activeOpacity={.8}
                    style={styles.button}>
                        <Text style={styles.plus}>Login</Text>
                </TouchableOpacity>
            </View>
        </View>
    )
}

const styles = StyleSheet.create({
    container: {
        width: width,
        height: height/12,
        backgroundColor: '#3D348B',
        flexDirection: 'row'
    },
    buttonContainer: {
        flex: 1,
        marginTop: 'auto',
        marginBottom: 10
    },
    buttonContainerLeft: {
        alignItems: 'flex-start',
    },
    buttonContainerRight: {
        alignItems: 'flex-end'
    },
    button: {
        // backgroundColor: '#E6AF2E',
        justifyContent: 'center',
        alignItems: 'center',
        // borderRadius:98,
        // width: 49,
        // height: 49,
        marginLeft: 24,
        marginRight: 24,
    },
    plus: {
        color: '#E0E2DB',
        fontSize: 24

    }
  });

export default HeaderBar;