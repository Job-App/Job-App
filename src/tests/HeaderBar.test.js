import React from 'react';
import { shallow } from 'enzyme';
import HeaderBar from '../components/HeaderBar/Headerbar.js';

test('Testing HeaderBar component', () => {
 const wrapper = shallow(<HeaderBar />);
 expect(wrapper).toMatchSnapshot();
});