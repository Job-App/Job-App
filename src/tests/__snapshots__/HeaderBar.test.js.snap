// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing HeaderBar component 1`] = `
<View
  style={89}
>
  <View
    style={
      Array [
        90,
        91,
      ]
    }
  >
    <TouchableOpacity
      activeOpacity={0.8}
      style={93}
    >
      <Text
        style={94}
      >
        Add
      </Text>
    </TouchableOpacity>
  </View>
  <View
    style={
      Array [
        90,
        92,
      ]
    }
  >
    <TouchableOpacity
      activeOpacity={0.8}
      style={93}
    >
      <Text
        style={94}
      >
        Login
      </Text>
    </TouchableOpacity>
  </View>
</View>
`;
